src/services/utils.js

```diff
@@ -135,16 +135,24 @@ const downloadFile = (blob, filename, type) => {
 };
 
 function fetchEnv() {
-    return fetch('env.json').then((res) => res.json());
+    return fetch('env.json').then((res) => {
+        return res.json();
+    });
+}
+
+export function fetchIdpSettings() {
+    return fetch('idpSettings.json');
 }
 
 export function fetchAuthorizationCodeFlowFeatureFlag() {
     console.info(`Fetching authorization code flow feature flag...`);
     return fetchEnv()
-        .then((env) =>
-            fetch(env.appsMetadataServerUrl + '/authentication.json')
-        )
-        .then((res) => res.json())
+        .then((env) => {
+            return fetch(env.appsMetadataServerUrl + '/authentication.json');
+        })
+        .then((res) => {
+            return res.json();
+        })
```

Comment:
you can restore original shorter form if you return with only one statement
```suggestion
        .then((env) => fetch(env.appsMetadataServerUrl + '/authentication.json'))
        .then((res) => res.json())
```